cli/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'bio.terra.pfb.java-common-conventions'
+}
+
+version 'unspecified'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'info.picocli:picocli:4.7.4'
+}
+
+jar {
+    manifest {
+        attributes 'Main-Class': 'bio.terra.pfb.JavaPfbCommand'
+    }
+    archiveBaseName.set('java-pfb-cli')
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+    }
+}
+
+test {
+    useJUnitPlatform()
+}
```

Comment:
You may have copied this from my project (or the template) but I don't think it's needed if you also use

```
tasks.named('test') {
    useJUnitPlatform()
}
```

I'd try removing this from here and `library/build.gradle` and see if the junit tests still run.